ghost/core/core/server/data/migrations/versions/5.45/2023-04-17-11-05-add-post-revision-author.js

```diff
@@ -0,0 +1,7 @@
+const {createAddColumnMigration} = require('../../utils');
+
+module.exports = createAddColumnMigration('post_revisions', 'created_by', {
+    type: 'string',
+    maxlength: 24,
+    nullable: true
```

Comment:
Set this to `nullable` to not interfere with existing records

Reply:
👍